overlord/devicestate/devicestate_install_api_test.go

```diff
@@ -243,8 +281,32 @@ func (s *deviceMgrInstallAPISuite) testInstallFinishStep(c *C, opts finishStepOp
 
 	// Insert encryption data when enabled
 	if opts.encrypted {
+		// Mock TPM and sealing
+		restore := devicestate.MockSecbootCheckTPMKeySealingSupported(func() error { return nil })
+		s.AddCleanup(restore)
+		restore = boot.MockSealKeyToModeenv(func(key, saveKey keys.EncryptionKey, model *asserts.Model, modeenv *boot.Modeenv, flags boot.SealKeyToModeenvFlags) error {
+			return nil
```

Comment:
question: does the test quality improves if we verify that these functions are called, and with the right params?

Reply:
I agree that maybe this should check the arguments a bit more

Reply:
Added some additional checks